api/config/service/db/postgres/v1/database.proto

```diff
@@ -32,4 +32,6 @@ message Connection {
 // TODO: Expose more database/sql tunables.
 message Config {
   Connection connection = 1;
+  // The default max idle connections is currently 2 and cannot be configured lower
+  int32 max_idle_connections = 2 [ (validate.rules).int32 = {gt : 2} ];
```

Comment:
i would make this an optional value. 0 is a valid value. though negative values are the same as zero. so could also do `if 0 default; else val`